content/v1.11-docs/projects/istio-csr.md

```diff
@@ -32,12 +32,12 @@ Running istio-csr requires a few steps and preconditions in order:
 3. An `Issuer` or `ClusterIssuer` which will be used to issue Istio certificates
 4. istio-csr installed (likely via helm)
 5. Istio [installed](https://istio.io/latest/docs/setup/install/istioctl/) with
-   some custom config required, e.g. using the example config from the [repository](https://github.com/cert-manager/istio-csr/tree/main/hack).
+   some custom config required, e.g. using the example config from the [repository](https://github.com/cert-manager/istio-csr/tree/main/make/config/istio).
```

Comment:
suggestion: It's great that we're fixing this and this is obviously an improvement.

While we're here, could we not use `main`? That's the reason this broke in the first place. If we linked to a specific revision in the docs it wouldn't auto update but it also wouldn't auto break. I'd rather have it not automatically break in the future